run.py

```diff
@@ -59,57 +69,59 @@
 print("Extracting Features...")
 # Retrieve db rows and store as a dataframe
 df = dbHelper.getRowsAsDf("SELECT * from clips where url = '{:}' ORDER BY start ASC".format(vidUrl))
+# Drop null columns with features to be populated
+feature_list = ['mfcc', 'amplitude', 'pitch', 'word', 'subjectivity', 'polarity', 'pred_excitement', 'pred_highlight_rf', 'pred_highlight_nn']
```

Comment:
Why do we drop columns?

Reply:
we don't need them until we overwrite them - and it breaks concat hehe.

Reply:
Another reason why we shouldn't drop: If something breaks in the classification but we've already written all the feature data to the db, we can use line 72 to grab that data, skip feature extraction, and run classification